styleguide/source/_patterns/02-molecules/partner-promo.twig

```diff
@@ -0,0 +1,12 @@
+{% set image = partnerPromo.image %}
+<div class="ama__partner-promo">
+  <div class="ama__partner-promo__image">
+    {% include '@atoms/image/image.twig' %}
+  </div>
+  <div class="ama__partner-promo__text">
+    <h2 class="ama__partner-promo__heading">{{ partnerPromo.heading }}</h2>
```

Comment:
Can we confirm that we need a custom heading here instead of using the heading atom?